go.mod

```diff
@@ -1,6 +1,9 @@
-module github.com/hashicorp/go-immutable-radix
+module github.com/hashicorp/go-immutable-radix/v2
+
+go 1.18
 
 require (
 	github.com/hashicorp/go-uuid v1.0.0
-	github.com/hashicorp/golang-lru v0.5.0
+	github.com/hashicorp/golang-lru v0.5.4
```

Comment:
could we update this dependency to the v2 version of golang-lru? (this one introduces generics for the cache as well)

Reply:
good catch, done!